### PR TITLE
[MRG] ENH: Some examples updates to plot_matrix

### PR DESCRIPTION
**Pull request by KamalakerDadi**
Updating with our new matrix plotting function ```nilearn.plotting.plot_matrix``` in following examples:
 - examples/02_decoding/plot_haxby_multiclass.py
 - examples/03_connectivity/plot_group_level_connectivity.py
 - examples/03_connectivity/plot_multi_subject_connectome.py
 
This was discussed last week IRL with @GaelVaroquaux

### CODE REVIEW

**Comment by bthirion** [comment]
LGTM. Beware the Travis failure.

**Comment by KamalakerDadi** [comment]
>Beware the Travis failure.

Travis should FIX with PR #1493

File: examples/02_decoding/plot_haxby_multiclass.py [hunk 1]
```diff
@@ -98,21 +98,20 @@
...
 svc_ova.fit(X[session < 10], y[session < 10])
 y_pred_ova = svc_ova.predict(X[session >= 10])
 
-plt.matshow(confusion_matrix(y_pred_ova, y[session >= 10]))
-plt.title('Confusion matrix: One vs All')
-plt.xticks(np.arange(len(unique_conditions)), unique_conditions)
-plt.yticks(np.arange(len(unique_conditions)), unique_conditions)
+plot_matrix(confusion_matrix(y_pred_ova, y[session >= 10]),
+            labels=unique_conditions,
+            title='Confusion matrix: One vs All', cmap='coolwarm')
```
Comment (GaelVaroquaux) [inline examples/02_decoding/plot_haxby_multiclass.py]: The coolwarm cmap is not well adapted here: it's a symmetrical one, while the matrix goes from 0 to 1. We should use something like the "hot_r" (the hot, but "0" being white).

**Comment by GaelVaroquaux** [comment]
LGTM, aside the comment on the colorbar, which must be changed.

**Comment by GaelVaroquaux** [comment]
Colormap, I meant, not colorbar.

**Comment by GaelVaroquaux** [comment]
LGTM. Merging. Thanks!

**Comment by KamalakerDadi** [comment]
Thanks for merging.

**Pull request merged**